frontend/src/app/user/pc-member/shared/mock-members.ts

```diff
@@ -0,0 +1,6 @@
+import {PcMember} from './pcMember.model';
+
+export const MEMBERS: PcMember[] = [
```

Comment:
same here, use a service to mock stuff: a UserService that can fetch users and pc members too.